packages/legend-extension-dsl-serializer/src/models/__tests__/TEST_DATA_DSLSerializer_Roundtrip.ts

```diff
@@ -0,0 +1,124 @@
+/**
+ * Copyright (c) 2020-present, Goldman Sachs
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+export const roundtripTestData = [
+  {
+    path: 'anything::text',
```

Comment:
probably should update to remove `text` in these names. You copied the code from `DSL text`, but should try to update these to lessen confusion of future code readers